### ISSUE TITLE
Store comparison "View in" setting should use regional prices instead of plain conversion

### ISSUE THREAD

**Issue by snakecase** [tfedor/Enhanced_Steam#52]
Self-explanatory:
![screenshot](https://user-images.githubusercontent.com/4579891/52816486-2dffb700-30c3-11e9-96c3-20bdf74ed034.png)

**Comment by tfedor** [comment]
This seems like a problem with incorrect country identification. In your version it's checked from steamCountry cookie. I added check for cc param now as well. This may be some other problem though - could you check your steamCountry cookie first?

**Comment by snakecase** [comment]
Tested both settings with clean cookies before each try.
When the setting is set to Auto-Detect it automatically sets steamCountry to
![steamcountry](https://user-images.githubusercontent.com/4579891/52831086-1d683480-30f5-11e9-8747-67715c325e45.png)
When I set it to RUB the cookie doesn't get set at all and it still shows prices wrong.

I get it auto-detect gets confused since I'm currently in another region than my Steam account is set to but why doesn't it work when I specify which currency to use?
